recipes/semi-ate-stil/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "semi-ate-stil" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Semi-ATE-STIL-{{ version }}.tar.gz
+  sha256: ee1d08ad52b589059d04e66d2c2ec9d3947846bfb13cb225f51275516472db52
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - lark
+  run:
+    - python >=3.6
```

host and run should have the same lower limit.

```suggestion
    - python >=3.7
```